pincer/objects/guild/guild.py

```diff
@@ -699,18 +696,15 @@ async def edit_role(
         ...
 
     async def edit_role(
-        self,
-        id: Snowflake,
-        reason: Optional[str] = None,
-        **kwargs
+        self, id: Snowflake, reason: Optional[str] = None, **kwargs
```

Comment:
type `**kwargs`?

Reply:
@trag1c implemented that

Reply:
I realize that there is no need for any types on that function. It is already an overloaded function which means that there is a function with the types already there.